relnotes/taskext.feature.md

```diff
@@ -0,0 +1,67 @@
+## Built-in task and scheduler support in DaemonApp
+
+`ocean.util.app.DaemonApp` `ocean.util.app.ext.TaskExt`
+
+It is now possible to make `DaemonApp` initialize scheduler automatically using
+standard app configuration file. To do so, it is sufficient call
+`task_ext.enable` in the constructor:
+
+```D
+class MyApp : DaemonApp
+{
+    this ( )
+    {
+        super("name", "desc", VersionInfo.init);
+        this.task_ext.enable(); // NB: must be after `super` call
+    }
+```
+
+This extension is disabled by default to avoid clashing with existing
+application code which is already handling scheduler initialization in some way.
+
+If `enable` was called, overriden `run` method will be called within a task
+context and there is no need to create one manually or start scheduler event
+loop:
+
+```D
+class MyApp : DaemonApp
+{
+  this ( )
+  {
+    super("name", "desc", VersionInfo.init);
+    this.task_ext.enable();
+  }
+
+  override int run ( Arguments args, ConfigParser config )
+  {
+    assert(Task.getThis() !is null);
+    assert(isSchedulerUsed());
```

Comment:
Shouldn't we use `verify()` instead for this cases?

Reply:
Interesting question. In actual app code you shouldn't have any of these checks as they are 100% guaranteed to be true and present here only for demonstration. And for example purpose I use `assert` to minimize amount of assumed imported context. Don't feel particularly strong either way though.

Reply:
I see your points. 
(It might also be good to promote the usage of `verify()` through the examples though)